swagger/public_v1/component_schemas/ProviderAttributes.yml

```diff
@@ -0,0 +1,119 @@
+---
+type: object
+required:
+  - code
+  - name
+properties:
+  accredited_body?:
+    type: boolean
+    description: "Is this provider an accredited body for other provider's courses."
+    example: "true"
+  city:
+    type: string
+    description: "Town or city"
+    example: "London"
+  county:
+    type: string
+    description: "County"
+    example: "London"
+  changed_at:
+    type: string
+    format: date-time
+    description: |
+      Date-time timestamp of when this provider or any of its related data changed.
+    example: "2019-06-13T10:44:31Z"
+  code:
+    type: string
+    description: |
+      Code that uniquely identifies this provider within a recruitment cycle.
+    maxLength: 3
+    minLength: 3
+    example: X99
+  created_at:
+    type: string
+    format: date-time
+    description: "Timestamp of when this provider was created."
+    example: "2019-06-13T10:44:31Z"
+  # latitude:
```

Comment:
We should remove these and put them back if we decide we're able to provide them.

Reply:
Move to #1285